pkg/util/gpg.go

```diff
@@ -167,9 +208,22 @@ func getPublicKeyFromServer(id string, uri string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	if strings.Contains(strings.ToLower(string(b)), "no results found") {
+	if noPublicKeyFoundRx.MatchString(strings.ToLower(string(b))) {
 		return nil, errNoSuchPublicKey
 	}
 
 	return b, nil
 }
+
+func uploadPublicKey(cntr *schwift.Container, b []byte) ([]byte, error) {
+	if cntr == nil {
+		return b, nil
+	}
+	n := fmt.Sprintf("%x.asc", sha256.Sum256(b))
+	obj := cntr.Object(n)
+	err := obj.Upload(bytes.NewReader(b), nil, nil)
+	if err == nil && LogIndividualTransfers {
+		logg.Info("transferring to %s", obj.FullName())
```

Comment:
The log message is technically correct, but could be more explicit about the upload going to the cache rather than any of the job targets:
```suggestion
		logg.Info("transferring GPG key to cache at %s", obj.FullName())
```